src/cript/nodes/primary_nodes/material.py

```diff
@@ -71,6 +72,7 @@ class JsonAttributes(PrimaryBaseNode.JsonAttributes):
         identifiers: List[dict[str, str]] = field(default_factory=dict)
         # TODO add proper typing in future, using Any for now to avoid circular import error
         component: List["Material"] = field(default_factory=list)
+        process: Union[Process, None] = None
```

Comment:
ooh for this you want to use [Optional](https://docs.python.org/3/library/typing.html#typing.Optional)

`Optional` is pretty much equal to `Union[something, None]` 
> I have this in a bunch of places that I should remove later because `Union` is supposed to be used only if it is `" you can pass either this or that"` `Union[object1, object2]` instead of `Optional` if something can be there or not

leaned all about typing when I was working with mypy!

Reply:
if you submit another PR changing this, I can quickly approve it!

Reply:
sure, happy to learn here.

This is done.